architecture/additional_concepts/storage.adoc

```diff
@@ -13,15 +13,23 @@ toc::[]
 == Overview
 
 Managing storage is a distinct problem from managing compute resources.
-OpenShift leverages the Kubernetes `*PersistentVolume*` subsystem, which
-provides an API for users and administrators that abstracts details of how
-storage is provided from how it is consumed. This subsystem uses the
-`*PersistentVolume*` and `*PersistentVolumeClaim*` API objects.
-
-A `*PersistentVolume*` (PV) object represents a piece of existing networked
-storage in the cluster that has been provisioned by an administrator. It is a
-resource in the cluster just like a node is a cluster resource. PVs are volume
-plug-ins like `*Volumes*`, but have a lifecycle independent of any individual
+OpenShift leverages the Kubernetes persistent volume (PV) framework to allow
+administrators to provision persistent storage for a cluster. Using persistent
+volume claims (PVCs), developers can request PV resources without having
+specific knowledge of the underlying storage infrastructure.
+
+PVCs are specific to a
+link:../../architecture/core_concepts/projects_and_users.html#project[project]
+and are created and used by developers. A PVC is required if a developer wants
+to use a PV. The PVs themselves, however, are not bound to a single project;
+they can be shared across the entire OpenShift cluster and claimed from any
+project.
```

Comment:
Updated to the following:

http://file.rdu.redhat.com/~adellape/031016/storage-sec-update/architecture/additional_concepts/storage.html#overview